content/docs/engine-dev/tools/project-manager/bootup-sequence/_index.md

```diff
@@ -0,0 +1,167 @@
+---
+linkTitle: Bootup Sequence
+title: Boot-up sequence
```

Comment:
Use "Bootup" here, no hyphen. This is similar to the convention of using "set up" (verb) vs "setup" (adjective).
```suggestion
title: Bootup Sequence
```